core/embed/rust/src/ui/component/text/op.rs

```diff
@@ -238,7 +239,9 @@ impl<T: StringType + Clone> OpTextLayout<T> {
 #[derive(Clone)]
 pub enum Op<T: StringType> {
     /// Render text with current color and font.
-    Text(T),
+    /// Bool signifies whether the text continues or not
+    /// (so we know whether to show ellipsis or not).
```

Comment:
see https://github.com/trezor/trezor-firmware/pull/3107#discussion_r1260868635

Reply:
Done in [6c0fd54](https://github.com/trezor/trezor-firmware/pull/3135/commits/6c0fd547b0a383aa6a1bf480d01e7a8bdd5fc27b)